skyportal/handlers/api/spectrum.py

```diff
@@ -566,17 +515,19 @@ def get(self, obj_id):
         obj = Obj.query.get(obj_id)
         if obj is None:
             return self.error('Invalid object ID.')
+        query_options = [
+            joinedload(Spectrum.instrument.name)
+            .joinedload(Spectrum.groups)
+            .joinedload(Spectrum.reducers)
+            .joinedload(Spectrum.observers)
+            .joinedload(Spectrum.owner)
+        ]
         spectra = Obj.get_spectra_readable_by(
```

Comment:
Needs to be updated to use the new access methods

Reply:
Now uses new access methods